src/main/java/net/dv8tion/jda/api/Permission.java

```diff
@@ -45,6 +45,7 @@
     BAN_MEMBERS(          2, true, false, "Ban Members"),
     NICKNAME_CHANGE(     26, true, false, "Change Nickname"),
     NICKNAME_MANAGE(     27, true, false, "Manage Nicknames"),
+    MODERATE_MEMBERS(    40, true, false, "Moderate Members"),
```

Comment:
```suggestion
    MODERATE_MEMBERS(    40, true, false, "Time out Members"),
```
The name (string) should be the readable name used in the client